src/Test.FunctionalTests.Sarif/TestData/Multitool/ValidateCommand/Inputs/SARIF1004.ExpressUriBaseIdsCorrectly_Valid.sarif

```diff
@@ -10,18 +10,18 @@
         }
       },
       "originalUriBaseIds": {
-        "PROJECT_ROOT": {
+        "REPOROOT": {
```

Comment:
>REPOROOT [](start = 9, length = 8)

*For now*, let's use these:

- The old `PROJECT_ROOT` becomes `REPO_ROOT` (still contains an underscore).
- The old `SOURCE_ROOT` is unchanged (do _not_ change it to `SRCROOT`).
- You're right, `RULES_ROOT` is not in our current set. Changing to `TEST_ROOT` in this test file is fine -- but do use the underscore.